packages/vscode-extension/package.json

```diff
@@ -71,7 +71,14 @@
 			{
 				"command": "suitecloud.uploadfile",
 				"key": "ctrl+alt+u",
-				"mac": "ctrl+alt+u"
+				"mac": "ctrl+alt+u",
+				"when": "resource =~ //FileCabinet/SuiteScripts/.*\\.js$|^file$/"
```

Comment:
I don't know how exactly this "regex" works, but it looks like we won't be able to upload files from SuiteApps, because they are normally located at `FileCabinet/SuiteApps/AppId/myfile.txt` .  I also see that are other folder created in the FileCabinet that we should also allow. I'm not sure if with this regex we are allowing all types of files to be uploaded or just `.js`. Please check it.

Reply:
FileCabinet may contain .pdf .html .template and images etc I suppose

Reply:
FileCabinet can contain any kind of file.